src/game/server/gamecontext.cpp

```diff
@@ -4653,5 +4653,5 @@ void CGameContext::OnUpdatePlayerServerInfo(char *aBuf, int BufSize, int ID)
 		"\"team\":%d",
 		aJsonSkin,
 		JsonBool(m_apPlayers[ID]->IsAfk()),
-		m_apPlayers[ID]->GetTeam());
+		m_apPlayers[ID]->GetTeam() == -1 ? -1 : GetDDRaceTeam(ID));
```

Comment:
```suggestion
		m_apPlayers[ID]->GetTeam() == TEAM_SPECTATORS ? -1 : GetDDRaceTeam(ID));
```

The other -1 shouldn't be replaced.

Reply:
Interesting 🤔